apps/cookbook/src/app/examples/styling-list-example/styling-list-example.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="styling-list-example">
+  <h2>Unordered List Example</h2>
+  <ul>
+    <li>Aenean id odio turpis</li>
+    <li>Lorem ipsum dolor sit amet</li>
+    <li>Donec ullamcorper, risus id venenatis</li>
+  </ul>
+  <h2>Unordered List /w Nested Lists Example</h2>
+  <ul>
+    <li>Maecenas purus sapien</li>
+    <li>Pellentesque ultricies mi</li>
+    <li>
+      Dignissim et lorem mollis
+      <ul>
+        <li>Duis eget tincidunt sapien</li>
+        <li>Phasellus viverra ipsum sed</li>
+        <li>
+          Donec pulvinar aliquet turpis
+          <ul>
+            <li>Praesent vitae augue sed</li>
+            <li>Vitae convallis enim lobortis</li>
+            <li>Nisi condimentum ultricies nec</li>
+          </ul>
+        </li>
+      </ul>
+    </li>
+  </ul>
+  <h2>Ordered List Example</h2>
+  <ul>
+    <li>Nulla cursus sem nulla</li>
+    <li>Sed egestas lorem pharetra</li>
+    <li>Pellentesque habitant morbi</li>
+  </ul>
```

Comment:
Heading says "Ordered" but list is unordered
```suggestion
  <h2>Ordered List Example</h2>
  <ol>
    <li>Nulla cursus sem nulla</li>
    <li>Sed egestas lorem pharetra</li>
    <li>Pellentesque habitant morbi</li>
  </ol>
```